incubator/spring-cloud-data-flow/values.yaml

```diff
@@ -22,6 +35,24 @@ server:
   #  requests:
   #    cpu: 0.5
   #    memory: 640Mi
+
+skipper:
+  image: springcloud/spring-cloud-skipper-server
+  version: 1.0.1.RELEASE
+  imagePullPolicy: IfNotPresent
+  platformName: minikube
+  service:
+    type: ClusterIP
+    externalPort: 80
+    internalPort: 7577
```

Comment:
It probably doesn't make much sense to make container poprts configurable. Changing them would break things unless the Spring config is updated as well.

Reply:
Removing most of the port configs, only leaving the main server port as configurable